examples/components/singlepageviewer.js

```diff
@@ -0,0 +1,63 @@
+/* Copyright 2014 Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+if (!PDFJS.PDFViewer || !PDFJS.getDocument) {
+  alert('Please build the pdfjs-dist library using\n' +
+        '  `gulp dist-install`');
+}
+
+// The workerSrc property shall be specified.
+//
+PDFJS.workerSrc = '../../node_modules/pdfjs-dist/build/pdf.worker.js';
+
+// Some PDFs need external cmaps.
+//
+// PDFJS.cMapUrl = '../../node_modules/pdfjs-dist/cmaps/';
+// PDFJS.cMapPacked = true;
+
+var DEFAULT_URL = '../../web/compressed.tracemonkey-pldi-09.pdf';
+var SEARCH_FOR = ''; // try 'Mozilla';
+
+var container = document.getElementById('viewerContainer');
+
+// pdfLinkService not used since single page viewer
```

Comment:
This isn't correct, the whole point of the `PDFSinglePageViewer` (as opposed to `PDFPageView`) is that it supports the *same* interface as `PDFViewer` (for e.g. PDF files containing links).

Basically, this file should be an *almost* verbatim copy of https://github.com/mozilla/pdf.js/blob/master/examples/components/simpleviewer.js, but using `PDFJS.PDFSinglePageViewer` instead of `PDFJS.PDFViewer` and `var pdfSinglePageViewer` instead of `var pdfViewer`.